mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule PhoenixSwagger.Mixfile do
   use Mix.Project
 
-  @version "0.8.0"
```

Comment:
I'm not sure we need to bump to `0.9.0` for this change, unless there are some incompatibilities from the previous version? I'd assume `0.8.1` would be fine.

Reply:
If I remember, my reasoning behind this version change was that the swagger-ui repo had bumped their middle semver number also.